Dockerfile

```diff
@@ -1,41 +1,40 @@
-# This docker container can be pulled from umaprotocol/voting on dockerhub.
-# To get the latest image, run:
-#   docker pull umaprotocol/voting
-#
-# Execute the voting system with:
-#   docker run
-#     --env SENDGRID_API_KEY=<key>
-#     --env NOTIFICATION_FROM_ADDRESS=<email address>
-#     --env NOTIFICATION_TO_ADDRESS=<email address>
-#     --env LOCALHOST=<ethereum host>
-#     --env LOCALPORT=<ethereum port>
-#     --env MNEMONIC=<mnemonic>
-#     --env COMMAND="while true; do $(npm bin)/truffle exec ./scripts/Voting.js --network=<your_network>; sleep 60; done"
-#     umaprotocol/voting
+# This docker container can be pulled from umaprotocol/protocol on dockerhub.
+# To get the latest image, run: docker pull umaprotocol/protocol
+# This docker container is used to access all components of the UMA ecosystem
+# including liquidator, disputors and monitor bots. Settings for these bots are
+# defined via enviroment variables. For example to run a liquidator bot run:
+# docker run --env MNEMONIC="<mnemonic>" \
+#     --env PAGERDUTY_API_KEY="<pager duty api key>" \
+#     --env PAGERDUTY_SERVICE_ID="< service id>" \
+#     --env PAGERDUTY_FROM_EMAIL="<from email>" \
+#     --env SLACK_WEBHOOK="<slack webhook>" \
+#     --env EMP_ADDRESS="<emp address>" \
+#     --env POLLING_DELAY="<update delay in ms>" \
+#     --env COMMAND="npx truffle exec ../liquidator/index.js --network mainnet_mnemonic" \
```

Comment:
nit: why npx over `$(npm bin)`? IIUC `npx` doesn't respect the versions in our `package.json`, right?